scss/_patterns_pricing-block.scss

```diff
@@ -0,0 +1,48 @@
+@import 'settings';
+
+@mixin vf-p-pricing-block {
+  .p-pricing-block,
+  .p-pricing-block--25-75,
+  .p-pricing-block--50-50 {
+    @extend %vf-grid-row-equal-height;
+
+    .p-pricing-block__tier {
+      &:before {
+        border-top: 1px solid $colors--theme--border-default;
+        content: '';
+        margin-bottom: calc($spv--small - 1px);
+        width: calc(100% + map-get($grid-gutter-widths, default));
+      }
+
+      &:last-child::before {
+        width: 100%;
+      }
+
+      @media (max-width: $threshold-4-6-col) {
+        grid-column: span 4;
+
+        &:not(:last-child)::before {
+          width: 100%;
+        }
+      }
+    }
+  }
+
+  .p-pricing-block--25-75 {
+    @media (min-width: $threshold-6-12-col) {
+      .p-pricing-block__tier:first-child {
+        grid-column: 3 / span 2;
+      }
+    }
+
+    @media (min-width: $threshold-4-6-col) and (max-width: $threshold-6-12-col) {
+      .p-pricing-block__tier:nth-last-child(2)::before {
+        width: 100%;
+      }
+    }
+  }
+
+  .p-pricing-block--50-50 .p-pricing-block__tier {
+    grid-column: span 4;
```

Comment:
`span ${grid-8-columns / 2}` would make it clearer that this is taking half width and would tie this to the grid settings nicely